pwa/components/repairers/registration/RegistrationTunnelUserInfo.tsx

```diff
@@ -0,0 +1,123 @@
+import {Box, Button, Grid, TextField, Typography} from '@mui/material';
+import PasswordInput from '@components/form/input/PasswordInput';
+import React, {ChangeEvent, useContext, useState} from 'react';
+import {RepairerRegistrationContext} from '@contexts/RepairerRegistrationContext';
+import {validateEmail} from '@utils/emailValidator';
+import {UserFormContext} from '@contexts/UserFormContext';
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import {useRouter} from 'next/router';
+
+export const RegistrationTunnelUserInfo = (): JSX.Element => {
+  const router = useRouter();
+  const {
+    firstName,
+    lastName,
+    email,
+    setStepOneCompleted,
+    setTunnelStep,
+    setFirstName,
+    setLastName,
+    setEmail,
+  } = useContext(RepairerRegistrationContext);
+  const {password} = useContext(UserFormContext);
+  const [emailError, setEmailError] = useState<boolean>(false);
+  const [emailHelperText, setEmailHelperText] = useState<string>('');
+
+  const handleChangeFirstName = (
+    event: ChangeEvent<HTMLInputElement>
+  ): void => {
+    setFirstName(event.target.value);
+  };
+
+  const handleChangeLastName = (event: ChangeEvent<HTMLInputElement>): void => {
+    setLastName(event.target.value);
+  };
+
+  const handleChangeEmail = (event: ChangeEvent<HTMLInputElement>): void => {
+    setEmail(event.target.value);
+    if (!validateEmail(event.target.value)) {
+      setEmailError(true);
+      setEmailHelperText('Veuillez entrer une adresse email valide.');
+    } else {
+      setEmailError(false);
+      setEmailHelperText('');
+    }
+  };
+
+  const handleNextStep = () => {
+    setTunnelStep('choice');
```

Comment:
Wy do you need this tunnel step state?

Reply:
I added these stpes to manage the registration tunnel navigation. Do you think I should remove every tunnelStep states and replace them by the boolean steps like "stepOneCompleted"?